docs/libcurl/opts/Makefile.inc

```diff
@@ -258,6 +258,8 @@ man_MANS =                                      \
   CURLOPT_PROXY_CAPATH.3                        \
   CURLOPT_PROXY_CRLFILE.3                       \
   CURLOPT_PROXY_KEYPASSWD.3                     \
+  CURLOPT_PROXY_ISSUERCERT.3                    \
+  CURLOPT_PROXY_ISSUERCERT_BLOB.3               \ 
```

Comment:
This line ends in "\ " but must end in "\" no quotes. Other than that looks fine will check CI after fix is committed.

Reply:
ok, it is fixed